### PR TITLE
groupByColumns to ColumnName

### PR DESCRIPTION
**Pull request by devinrsmith**

### CODE REVIEW

**Review by rcaudy — APPROVED** [review]

**Comment by devinrsmith** [comment]
Nightlies passing: https://github.com/devinrsmith/deephaven-core/actions/runs/2743755257

File: engine/table/src/main/java/io/deephaven/engine/table/impl/by/ChunkedOperatorAggregationHelper.java [hunk 1]
```diff
@@ -3,6 +3,7 @@
  */
 package io.deephaven.engine.table.impl.by;
 
+import io.deephaven.api.ColumnName;
```
Comment (rcaudy) [inline engine/table/src/main/java/io/deephaven/engine/table/impl/by/ChunkedOperatorAggregationHelper.java:6]: Delete unused import:
`import io.deephaven.engine.table.impl.select.SelectColumn;`

File: server/src/main/java/io/deephaven/server/table/ops/ComboAggregateGrpcImpl.java [hunk 2]
```diff
@@ -4,7 +4,9 @@
 package io.deephaven.server.table.ops;
 
 import com.google.rpc.Code;
+import io.deephaven.api.ColumnName;
```
Comment (rcaudy) [inline server/src/main/java/io/deephaven/server/table/ops/ComboAggregateGrpcImpl.java:7]: Many newly unused imports.

File: engine/table/src/main/java/io/deephaven/engine/table/impl/UpdateBy.java [hunk 3]
```diff
@@ -109,10 +110,9 @@ public static Table updateBy(@NotNull final QueryTable source,
             }
         }
 
+        // TODO: pass through ColumnName to impl instead
```
[the remark this answers is not in the document]
Comment (devinrsmith) [reply engine/table/src/main/java/io/deephaven/engine/table/impl/UpdateBy.java:113]: https://github.com/deephaven/deephaven-core/issues/2693

**Review by rcaudy — APPROVED** [review]

**Review by rcaudy — APPROVED** [review]

**Pull request merged**